rafthttp/http.go

```diff
@@ -126,6 +137,17 @@ func (h *streamHandler) ServeHTTP(w http.ResponseWriter, r *http.Request) {
 		return
 	}
 
+	w.Header().Set("X-Server-Version", version.Version)
+
+	if err := checkVersionCompability(serverVersion(r.Header), minClusterVersion(r.Header)); err != nil {
+		log.Printf("rafthttp: request ignored due to %v", err)
+		http.Error(w, errMsgIncompatibleVersion, http.StatusForbidden)
+		return
+	}
+	if err := checkClusterVersionCompatiblity(r.Header.Get("X-Cluster-Version")); err != nil {
```

Comment:
can you give me an example that this check fails when the check at line 142 passes?